src/Eshopworld.WorkerProcess/LeaseAllocator.cs

```diff
@@ -69,7 +69,11 @@ public async Task<ILease> AllocateLeaseAsync(Guid instanceId)
                     await Task.Delay(delay).ConfigureAwait(false);
 
                     // activate
-                    lease.LeasedUntil = GetLeaseUntil(_options.Value.LeaseInterval);
+                    var now = ServerDateTime.UtcNow;
+                    lease.Interval = TimeSpan.FromMilliseconds(_options.Value.LeaseInterval.TotalMilliseconds -
```

Comment:
i think this calculation should be extracted and shared with workerlease

Reply:
That I can easily do